### PR TITLE
hook up wasi to wasmer

### PR DESCRIPTION
**Pull request by MarkMcCaskey**

### CODE REVIEW

File: src/bin/wasmer.rs [hunk 1]
```diff
@@ -215,6 +218,19 @@ fn execute_wasm(options: &Run) -> Result<(), String> {
         )
     };
 
+    #[cfg(features = "wasi")]
+    let (_abi, import_object) = if wasmer_wasi::is_wasi_module(&module) {
+        (
+            InstanceABI::WASI,
+            wasmer_emscripten::generate_import_object(),
```
Comment (lachlansneff) [inline src/bin/wasmer.rs]: emscripten?

File: src/bin/wasmer.rs [hunk 2]
```diff
@@ -215,6 +218,16 @@ fn execute_wasm(options: &Run) -> Result<(), String> {
         )
     };
 
+    #[cfg(features = "wasi")]
+    let (_abi, import_object) = if wasmer_wasi::is_wasi_module(&module) {
+        (InstanceABI::WASI, wasmer_wasi::generate_import_object())
```
Comment (lachlansneff) [inline src/bin/wasmer.rs]: We need to pass in the args and envs. They are both `Vec<Vec<u8>>`.

Comment (MarkMcCaskey) [reply src/bin/wasmer.rs]: Cool, was just going off the function signature at the time

**Pull request merged**